File: rest-service/src/main/java/mb/lib/query/model/JobLinkCollection.java
package mb.lib.query.model;

import java.util.HashMap;
import java.util.List;
import java.util.Map;

import mb.lib.model.HashID;

public class JobLinkCollection
{
  public final Map<HashID, List<BlastJobLink>> byParentID = new HashMap<>();
  public final Map<HashID, List<BlastJobLink>> byChildID  = new HashMap<>();
}

File: rest-service/src/main/kotlin/mb/lib/query/model/JobLinkCollection.kt
package mb.lib.query.model

import org.veupathdb.lib.hash_id.HashID

data class JobLinkCollection(
  val byParentID: MutableMap<HashID, MutableList<BlastJobLink>> = HashMap(),
  val byChildID:  MutableMap<HashID, MutableList<BlastJobLink>> = HashMap(),
)
